dirty_cat/_feature_augmenter.py

```diff
@@ -90,6 +90,7 @@ def fit(self, main_table) -> "FeatureAugmenter":
                     f"Got column key {key!r}, "
                     "but column missing in the auxilliary table."
                 )
+        # What happens if the column names between two aux_tables is identical?
```

Comment:
Usually, when something needs a later rework, it's good practice to indicate them with `TODO` or `FIXME`, like so:
```suggestion
        # FIXME: What happens if the column names between two aux_tables is identical?
```